### PR TITLE
tweaks

### PR DESCRIPTION
**Pull request by wasade**
Making figures on barnacle in test now.

### CODE REVIEW

**Comment by wasade** [comment]
@mortonjt @squirrelo @ElDeveloper, possible for review/merge? just tweaks that have been found/necessary for processing on barnacle.

**Comment by mortonjt** [comment]
:+1:

File: scripts/make_phyla_plots_AGP.py [hunk 1]
```diff
@@ -87,14 +87,14 @@ def main(otu_table, mapping_data, cat_tables, output_dir, sample_type='fecal',
...
-        michael_pollan = '000007108.1075657'
+        michael_pollan = '10317.000007108'
         cat_list = ['You', 'Average', 'Similar Diet', ' Similar BMI',
                     'Same Gender', 'Similar Age', 'Michael Pollan']
         order = ['Sample', 'Average', 'DIET_TYPE', 'BMI_CAT', 'SEX',
                  'AGE_CAT', 'MP']
 
     elif sample_type == 'skin':
-        michael_pollan = '000007113.1075702'
+        michael_pollan = '10317.000007113'
```
Comment (ElDeveloper) [inline scripts/make_phyla_plots_AGP.py]: This is only due to the inconsistent sample names in the actual submissions right, not due to a Pandas problem?

Comment (wasade) [inline scripts/make_phyla_plots_AGP.py]: The change? It's necessary as the sample IDs have changed due to the new EBI submissions

**Comment by squirrelo** [comment]
:+1:

File: scripts/make_phyla_plots_AGP.py (hunk 1 above)
Comment (ElDeveloper) [inline scripts/make_phyla_plots_AGP.py]: OK, just checking!

**Pull request merged**